### DISCUSSION TITLE
Does all NeMo ASR models support in Riva?

### DISCUSSION THREAD

**Discussion by mehadi92** [NVIDIA-NeMo/NeMo#6861] [Q&A] [answered]
Does this models support in Riva
1. conformer model: [STT En Fast Conformer-CTC Large | NVIDIA NGC 1](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_en_fastconformer_ctc_large)
2. Transducer model: [STT En Fast Conformer-Transducer Large | NVIDIA NGC 1](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_en_fastconformer_transducer_large)

**Comment by VahidooX** [comment, marked as answer]
Currently all CTC models are supported in Riva while Transducer models not yet.

**Reply by titu1994** [reply]
Fast conformer CTC is not fully tested yet, but it might work

**Reply by mehadi92** [reply]
@titu1994 I have test `stt_en_fastconformer_ctc_large` model. when I deploy it in Riva it give only subword prediction like the bellow
```sh
python transcribe_file.py --input-file data/examples/en-US_sample.wav --server localhost:50052 --print-confidence
## is 
Confidence:   -0.6694
----
## ural 
Confidence:   -0.0005
----
## al 
Confidence:   -0.0001
----
## ing 
Confidence:   -0.0184
----
## ing 
Confidence:   -0.0000
----
## ing 
Confidence:   -0.0000
----
```

**Reply by mehadi92** [reply]
Also ask in the Riva forum they said it's not supported yet, [fastconformer models riva support status](https://forums.developer.nvidia.com/t/does-nemo-fastconformer-and-transducer-model-support-in-riva/256201) cc: @VahidooX

**Reply by titu1994** [reply]
Hmm interesting. They will eventually support it, NeMo models are usually exported in stages
